views/js/pciCreator/ims/mathEntryInteraction/creator/widget/states/Map.js

```diff
@@ -130,7 +130,7 @@ define([
             interaction = self.widget.element;
 
         interaction.onPci('responseChange', function (latex) {
-            if (self.inGapMode(self) === false && self.activeEditId !== null) {
+            if ((self.inGapMode(self) === false || self.inGapMode(self) === 'false') && self.activeEditId !== null) {
```

Comment:
**question:** Why not fix the `inGapMode` method instead?

https://github.com/oat-sa/extension-tao-itemqti-pci/blob/51e90a24a297bb3881bfb302ce0ee06d268b8bc0/views/js/pciCreator/ims/mathEntryInteraction/creator/widget/states/Map.js#L309

Reply:
**suggestion:** you could rather do something like this:

```javascript
    MathEntryInteractionStateResponse.prototype.inGapMode = function inGapMode() {
        var interaction = this.widget.element;
        var useGapExpression = interaction.prop('useGapExpression');
        return useGapExpression && useGapExpression !== 'false' || false;
    }
```